giskard-git/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:alpine
+
+RUN apk add --no-cache tini git \
+    && yarn global add git-http-server \
+    && adduser -D -g git git
+
+USER git
+WORKDIR /home/git
+
+RUN git init --bare repository.git
+
+RUN cd repository.git && git symbolic-ref HEAD refs/heads/main
+RUN git config --global user.email "giskard@example.com" && git config --global user.name "Giskard"
```

Comment:
actually this container should only be responsible for bare repo, so no need for cloning it. We may need to clone it in the "backend" container since it's the backend who will read/modify files in the "bare" repo